substrate/frame/ranked-collective/src/lib.rs

```diff
@@ -838,6 +846,93 @@ pub mod pallet {
 			Members::<T, I>::remove(&who);
 			Ok(())
 		}
+
+		/// Ensure the correctness of the state of this pallet.
+		#[cfg(any(feature = "try-runtime", test))]
```

Comment:
If you move this to a separate `impl` block it might look a bit neater, as we won't need to repeat the feature gate line.